spring-boot/admin/src/main/resources/templates/admin/fragments/profile/profile.html

```diff
@@ -66,8 +66,7 @@ <h2>Activity report</h2>
 
                             </div>
 
-                            <!--
-                            <div class="" role="tabpanel" data-example-id="togglable-tabs">
+                            <div th:if="${@environment.getProperty('ygg.admin.payable.enable')}" class="" role="tabpanel" data-example-id="togglable-tabs">
```

Comment:
I would suggest to move the `th:if="..."` into the tag in line 60 `<div class="col-md-9 col-sm-9 col-xs-9">`, because then the whole column will be hidden, also the heading "Activity report" will vanish (heading makes no sense if there is nothing below).